content/pages/home.md

```diff
@@ -77,5 +77,26 @@ blocks:
       We will make sure you get a quick response from our team if you share your
       phone number and email with us.
     _template: quoteSection
+  - heading: Join Our Team
+    careerDescription: >
+      ### Whether you’re a beginner or a CNC expert, there are
+      excitingopportunities for all experience levels at Kenworthy Machine.
```

Comment:
Small typo: There needs to be a space between 'exciting' and 'opportunities', I pushed a change to address this!